taskboot/cli.py

```diff
@@ -268,6 +269,23 @@ def main():
     )
     deploy_pypi.set_defaults(func=publish_pypi)
 
+    # Push on github
+    github_push_cmd = commands.add_parser(
+        "github-push", help="Push the commits of a branch on a GitHub repository",
```

Comment:
```suggestion
        "git-push", help="Push the commits of a branch on a Git repository",
```

I know right now it requires GitHub as it's using the GitHub API, but in theory we can make it agnostic, so let's use the generic name.

Reply:
And actually, we can make it agnostic right now by doing one of:
- add user login as a parameter of the command;
- get the user login using `git config user.email`;
- get user login from the secret.

I don't have a strong preference between the three options.

Reply:
> And actually, we can make it agnostic right now by doing one of:
> 
>     * add user login as a parameter of the command;
> 
>     * get the user login using `git config user.email`;
> 
>     * get user login from the secret.
> 
> 
> I don't have a strong preference between the three options.

I prefer adding the user login as a command line parameter, in this way we don't have additional dependencies :)